src/client/src/components/Meal.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Meal(params) {
+  const meal = params.meal;
```

Comment:
Again, params is not usual for React props. 
"props" or "meal" and then destructuring the object for its properties, please 🙏🏽